gulpfile.js

```diff
@@ -0,0 +1,18 @@
+var gulp = require('gulp');
+var shell = require('gulp-shell');
+var browserSync = require('browser-sync').create();
+
+
+// Task for building blog when something changed:
+gulp.task('build', shell.task([
+  'jekyll serve'
```

Comment:
Does this work?  I ask because I thought the `serve` command blocks until completion (which will never happen since it just sits there waiting forever. 

Also you're making build do a serve, why not just do that in the task below?